apps/andi/lib/andi_web/live/ingestion_live_view/edit_ingestion_live_view.ex

```diff
@@ -8,26 +8,56 @@ defmodule AndiWeb.IngestionLiveView.EditIngestionLiveView do
   alias Andi.Services.IngestionStore
   alias Andi.Services.IngestionDelete
   alias Andi.InputSchemas.InputConverter
+  alias AndiWeb.InputSchemas.IngestionMetadataFormSchema
+  alias Ecto.Changeset
 
   import SmartCity.Event, only: [ingestion_update: 0]
 
   @instance_name Andi.instance_name()
 
   access_levels(render: [:private])
 
+  def mount(
+        _params,
+        %{"is_curator" => is_curator, "ingestion" => ingestion, "user_id" => user_id} = _session,
+        socket
+      ) do
+    default_changeset = InputConverter.andi_ingestion_to_full_ui_changeset(ingestion)
+
+    {:ok,
+      assign(socket,
+        changeset: default_changeset,
+        click_id: nil,
+        delete_ingestion_modal_visibility: "hidden",
+        unsaved_changes_modal_visibility: "hidden",
+        is_curator: is_curator,
+        unsaved_changes: false,
+        page_error: false,
+        ingestion: ingestion,
+        save_success: false,
+        success_message: "",
+        user_id: user_id
+      )}
+  end
+
   def render(assigns) do
+    metadata_changeset = IngestionMetadataFormSchema.extract_from_ingestion_changeset(assigns.changeset)
+    ingestion_published? = assigns.ingestion.submissionStatus == :published
+
     ~L"""
     <%= header_render(@is_curator, AndiWeb.HeaderLiveView.header_ingestions_path()) %>
     <div class="edit-page" id="ingestions-edit-page">
-      <%= f = form_for @changeset, "" %>
-        <%= hidden_input(f, :sourceFormat) %>
         <div class="edit-ingestion-title">
           <h1 class="component-title-text">Define Data Ingestion</h1>
         </div>
 
-        <div>
-          <%= live_render(@socket, AndiWeb.IngestionLiveView.MetadataForm, id: :ingestion_metadata_form_editor, session: %{"ingestion" => @ingestion}) %>
-        </div>
+      <div>
+        <%= live_component(@socket, AndiWeb.IngestionLiveView.MetadataForm,
```

Comment:
Changing child to a live_component so that:
- It shares a process with the parent, eliminating possibility of race conditions
- Reduces complexity of pub/sub tracing
- Enables re-rendering of child when passing state down